### PR TITLE
Card Video (#26)

### LINKED ISSUE #26: Component - Card_Video
**Widget**: Card_Video

**Image**:
Card_Video

**Dimensions/Styles**: https://zpl.io/bAyEvW6

**Inputs**
- Image: For the background
- String: Label
- `Play_Button`: Reused widget

**Notes**:
- Should have a layer on top of the image (?)
- Depends on the implementation of `Play_Button`

### PR DESCRIPTION
**Pull request by miguelpruivo**
Adds `CardVideo` that takes background URL (image) as required parameter and, optionally, a label which can be positioned using the `labelAlignment` property (defaults to `Alignment.bottomCenter`).

Based on design, the label can have multiple locations, see [here](https://app.zeplin.io/project/5e711e6973c497b6352ed9f3/screen/5e73b66e1285030494b00dee) and [here](https://zpl.io/a38lzR1), thus, the `labelAlignment` property.

ezgif com-video-to-gif

Closed #26.

### CODE REVIEW

File: lib/resources/style/themes.dart [hunk 1]
```diff
@@ -44,8 +44,8 @@ class Themes {
       title: TextStyles.h2(
         color: colorText,
       ),
-      subtitle: TextStyles.paragraphSmall(
-        color: colorTextLight,
+      subtitle: TextStyles.subtitle(
```
Comment (Vanethos) [inline lib/resources/style/themes.dart]: Why is this changed?

File: lib/ui/screens/home/components/card_video.dart [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    : Theme.of(context).disabledColor,
+              ),
+            ),
+          ),
+          if (label != null)
+            Align(
+              alignment: labelAlignment,
+              child: Text(
+                label,
+                style: Theme.of(context).textTheme.subtitle.copyWith(shadows: [
```
Comment (Vanethos) [inline lib/ui/screens/home/components/card_video.dart]: It should use `h5`, not `subtitle`, see Zeplin and `text_styles.dart`

File: lib/ui/screens/home/components/card_video.dart [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            highlightColor: Colors.transparent,
+            child: RaisedButton(
+              elevation: 4.0,
+              onPressed: () {},
+              disabledColor: Colors.transparent,
+              child: Icon(
+                Icons.play_circle_filled,
+                size: 48.0,
+                color: onPressed == null
+                    ? Theme.of(context).textTheme.button.color
```
Comment (Vanethos) [inline lib/ui/screens/home/components/card_video.dart]: Shouldn't it always be with the primary color?

File: lib/resources/style/themes.dart (hunk 1 above)
Comment (miguelpruivo) [reply lib/resources/style/themes.dart]: Because I need pick subtitle like mentioned in the design and I see that there's a `TextStyles.subtitle` which IMO should match `subtitle` property, for naming consistency purposes. 😃

File: lib/ui/screens/home/components/card_video.dart (hunk 3 above)
Comment (miguelpruivo) [reply lib/ui/screens/home/components/card_video.dart]: Since we are using an icon as a Raised Button button here, it won't have a different color if enabled/disabled. This will trigger that change.

File: lib/resources/style/themes.dart (hunk 1 above)
Comment (Vanethos) [reply lib/resources/style/themes.dart]: But in this case, can't we refer it directly from the TextStyles? because it's impossible that every style can be placed inside TextTheme

File: lib/ui/screens/home/components/card_video.dart (hunk 3 above)
Comment (Vanethos) [reply lib/ui/screens/home/components/card_video.dart]: ok!
